src/libraries/Common/src/System/Net/Http/aspnetcore/Http3/QPack/H3StaticTable.cs

```diff
@@ -39,21 +38,13 @@ internal class H3StaticTable
             [HttpMethod.Put] = 21,
         };
 
-        private H3StaticTable()
-        {
-        }
-
-        public static H3StaticTable Instance { get; } = new H3StaticTable();
-
-        public int Count => _staticTable.Length;
-
-        public HeaderField this[int index] => _staticTable[index];
```

Comment:
The indexers aren't allowed in static class. I tried to check if it was used any where but found none. However, I can't confirm because I run into build problems on my machine, so I'll wait for the CI build to finish and check.